src/components/Sidebar/index.jsx

```diff
@@ -139,6 +143,7 @@ const Sidebar = ({
       icon: <Icon src={CreditCard} />,
       hidden: !enableSubscriptionManagementScreen,
```

Comment:
When the Subscription Management isn't visible, we should also not show the Onboarding Flow for it.